### PR TITLE
Wasabi+1 network upgrade proposal

### PR DESCRIPTION
**Pull request by diega**

### CODE REVIEW

File: IPs/RSKIP141.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- Add EXTCODEHASH opcode
+- Add CHAINID opcode (ref. [EIP-1344](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1344.md))
+- 2WP Locking Cap
+- 2WP Onchain Events
+- ValidateEthPow precompiled contract
+
+### Tentatively Accepted EIPs
+
+- ECADD, ECMUL and Pairing check precompiled contract
+- SELF_BALANCE opcode and SLOAD, BALANCE and EXTCODEHASH cost reduction (ref. [EIP-1884](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1884.md))
```
Comment (SergioDemianLerner) [inline IPs/RSKIP141.md]: Most are cost increases rather than cost reductions. only SELF_BALANCE is a reduction.

File: IPs/RSKIP141.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- 2WP Segwit Compatibility
+- Disable TXINDEX, DUPN, SWAPN, OP_HEADER opcodes
+- CalculateMMR precompiled contract
```
Comment (SergioDemianLerner) [inline IPs/RSKIP141.md]: As I remember, having a precompiled contract for this was an option that was discarded. The MMR would be computed by native code, right?

File: IPs/RSKIP141.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+### Proposed EIPs
+
+- Decentralize feePerKb calculation for precompiled Bridge contract
+- Costs reductions (ref. [EIP-1108](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1108.md) / [EIP-2028](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2028.md) / [EIP-2200](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2200.md))
+- Add Blake2 compression function F precompile (ref. [EIP-152](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-152.md))
+- GetRefund precompiled contract (ref. [RSKIP-139](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP139.md))
+- Add internal transactions hash in block header
+- Add bloom filter hash in block header
+- Precompiled Bridge contract events in Solidity format
+
```
Comment (SergioDemianLerner) [inline IPs/RSKIP141.md]: We also mentioned replacing Bloom value with hash(bloom) when computing the transaction id (all the RPC interface remains equal). The objective is to reduce FlyClient proofs (bloom takes 256 bytes, which are totally unnecessary) and eventually reduce storage for nodes that do not care about bloom

File: IPs/RSKIP141.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+### Included RSKIPs
+
+- TBD
+
+### Accepted RSKIPs
+
+- Add EXTCODEHASH opcode
+- Add CHAINID opcode (ref. [EIP-1344](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1344.md))
+- 2WP Locking Cap
```
Comment (josedahlquist) [inline IPs/RSKIP141.md]:
```suggestion
- [2WP Locking Cap](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP134.md)
```

**Pull request merged**